test/orchard/xref_test.clj

```diff
@@ -52,7 +55,7 @@
   (testing "basics"
     (let [expected #{#'orchard.xref-test/fn-deps-test #'orchard.xref-test/fn-dep #'clojure.core/even?
                      #'clojure.core/filter #'orchard.xref-test/fn-transitive-dep #'clojure.core/map
-                     #'clojure.test/test-var #'clojure.core/range}]
+                     #'clojure.test/test-var #'clojure.core/range #'clojure.core/inc'}]
```

Comment:
It would help me to understand, why is `inc'` returned now and not before?

Aren't both `#'clojure.core/range` and `#'clojure.core/inc'` aot-compiled?

Reply:
That's one of the non-static functions in core, like `conj`. It gets called by `range` as you said, but only the 0 arity version. 
The interesting thing is that inc' has an `:inline` meta but because it doesn't call the function, just passes it to `iterate` it actually does it with the var.

There are two new learnings here: inline dependencies get found if they are just passed along, which does make sense and we always return all arity dependencies. I don't think we can fix the arity thing.

It might make sense to check in with Alex Miller at some point to see if he would be willing to make some modifications to the clojure compiler to make these reflection type things easier and more accurate.

Reply:
Thanks!

Please simply add a more explicit test just like a past one:

![image](https://user-images.githubusercontent.com/1162994/148667701-aa8223de-de0c-4bcc-b019-dfaf3e5d8ecf.png)

so that we can remember the special meaning inc' has here.